css/templates/lune/components/_accordion.css

```diff
@@ -1,3 +1,13 @@
 /* //////////////////////////////////////////////////////
    ACCORDION
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+amp-accordion section[expanded] .show-more {
+  display: none;
+}
+amp-accordion section:not([expanded]) .show-less {
+  display: none;
+}
```

Comment:
Taken inspiration from the docs https://ampbyexample.com/components/amp-accordion/#styling-the-accordion